src/rws_client.cpp

```diff
@@ -474,6 +474,30 @@ RWSClient::RWSResult RWSClient::setSpeedRatio(unsigned int ratio)
   return evaluatePOCOResult(httpPost(uri, content), evaluation_conditions);
 }
 
+RWSClient::RWSResult RWSClient::loadModuleIntoTask(const std::string& task, const FileResource& resource, const bool replace)
+{
+  std::string uri = generateRAPIDTasksPath(task) + "?" + Queries::ACTION_LOAD_MODULE;
+  std::string content = Identifiers::MODULEPATH + "=" + generateFilePath(resource) + "&replace=" + ((replace) ? "true" : "false");
```

Comment:
Path to the file should be a direct path, i.e. "{directory}/{filename}", but `generateFilePath(resource)` adds "fileservice/" at the beginning.